CONTRIBUTING-ACL2020.md

```diff
@@ -1,14 +1,132 @@
+
+Before you start, please make sure you have a GitHub account and have your ssh key added to your GitHub account. Also, Python should be installed into your device to setup the environment.
+
+### Get a GitHub Repository
+
+```shell
+cd /path/to/the/local/folder
+git clone git@github.com:acl-org/acl-2020-virtual-conference.git
+```
+
+By then, you will be able to download all content from ```acl-2020-virtual-conference``` repository to your local folder. If you are  not familiar with ```git```, you can also download the content by clicking the green button and the "Download ZIP" button and extract to your local folder.
+
+
+
+### Install Required Packages
+
+First, move to the repo folder:
+
+```shell
+cd acl-2020-virtual-conference
+```
+
+Then install required packages:
+
+``` shell
+pip install -r requirements.txt
+```
+
+Additionally, you can create a virtual Python environment before the installation. There are many ways to create virtual environments in Python, which the following commands will guide you to create one using ```virtualenvwrapper``` with Python 3.x.
+
+```shell
+#----------------------------------------------------
+# on Ubuntu
+sudo apt-get install python3-pip
+sudo pip3 install virtualenvwrapper
+#----------------------------------------------------
+# on MacOS
+# install python3 if needed
+brew install python3
+#----------------------------------------------------
+which python3 # find the path to python3.x
+which virtualenvwrapper.sh # find the path to the virtualenverapper installation script
+# change the bash script file to activate wrapper
+nano ~/.bashrc
+# in the ~/.bashrc file
+# modify the following lines accordingly and add them at the end of the file
+export VIRTUALENVWRAPPER_PYTHON='the output of which python3'
+source 'the output of which virtualenvwrapper.sh'
+# close ~./bashrc by pressing ctrl+x and save the changes by pressing "y"
+# refresh the bash profile
+source ~/.bashrc
+# create virtual environment for acl
+mkvirtualenv --python='the output of which python3' your_env_name
+# by now you will have a python3.x virtual environment specificaly for acl-2020
+# if you accidiently quit the current session,
+# you can reactivate the environemnt by the following code
+workon your_env_name
+# install required packages
+pip install -r requirements.txt
+```
+
+
+
+Now the environment is activated.
+
+
+
+## Additional Instructions
+
+
 
 Here are additional instructions for working on the ACL2020 fork.
+
 1. Run the command below check out the private [sitedata_acl2020](https://github.com/acl-org/acl-2020-virtual-conference-sitedata).
-    ```bash
-    git submodule update --init --recursive
-    ```
+
+   ```bash
+   git submodule update --init --recursive
+   ```
+
    * You need to clone with SSH instead of HTTPS to get the git submodule working.
    * All ACL2020 sitedate are stored in this folder.
+
 2. All images should be saved under [static/images/acl2020](./static/images/acl2020) folder.
```

Comment:
Create an `FAQ` section at the end, and put this there.


## FAQ

Q: Where should I put images for ACL2020?
A: All images should be saved under [static/images/acl2020](./static/images/acl2020) folder.